test/unit/logops_test.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-var logUtils = require('./log-utils');
+var logUtils = require('./log_utils');
```

Comment:
Minor: I've modified the underscore to keep file names consistent across the whole project.